test/functional/ui/tabpage_spec.lua

```diff
@@ -0,0 +1,38 @@
+local helpers = require('test.functional.helpers')(after_each)
```

Comment:
let's put this in `test/functional/normal/tabpage_spec.lua`. The /ui/ subdir is for actual UI tests, i.e. `screen:expect()` things. 

(Though /normal/ isn't really a great place for this either, but that's why I don't like all of these categories...)

Reply:
> let's put this in test/functional/normal/tabpage_spec.lua.

Fixed.

> The /ui/ subdir is for actual UI tests, i.e. screen:expect() things.

Noted!